src/xcode/ENA/ENAUITests/AntigenTestProfile/ENAUITests_12_AntigenTestProfile.swift

```diff
@@ -13,10 +13,10 @@ class ENAUITests_12_AntigenTestProfile: XCTestCase {
 		continueAfterFailure = false
 		app = XCUIApplication()
 		app.setDefaults()
-		app.launchArguments.append(contentsOf: ["-isOnboarded", "YES"])
-		app.launchArguments.append(contentsOf: ["-setCurrentOnboardingVersion", "YES"])
-		app.launchArguments.append(contentsOf: ["-antigenTestProfileInfoScreenShown", "NO"])
-		app.launchArguments.append(contentsOf: ["-removeAntigenTestProfile", "YES"])
+		app.launchArguments.append(contentsOf: ["-isOnboarded", YES])
+		app.launchArguments.append(contentsOf: ["-setCurrentOnboardingVersion", YES])
+		app.launchArguments.append(contentsOf: ["-antigenTestProfileInfoScreenShown", NO])
+		app.launchArguments.append(contentsOf: ["-removeAntigenTestProfile", YES])
```

Comment:
Unnecessary move but now it's in 🤷‍♂️

Reply:
Why is this unnecessary? I need to set the info screen to not shown so I can test it.